generator/internal/format_class_comments.h

```diff
@@ -23,7 +23,8 @@ namespace cloud {
 namespace generator_internal {
 
 std::string FormatClassCommentsFromServiceComments(
-    google::protobuf::ServiceDescriptor const& service);
+    google::protobuf::ServiceDescriptor const& service,
+    std::string const& service_name);
```

Comment:
It might be nice to leave a brief breadcrumb here about `service_name_mapping` to explain why `service_name` may not be the same as `service.name()`.

Reply:
Done